python/cudf/cudf/tests/test_orc.py

```diff
@@ -670,3 +674,28 @@ def test_orc_reader_gmt_timestamps(datadir):
     pdf = orcfile.read().to_pandas()
     gdf = cudf.read_orc(path, engine="cudf").to_pandas()
     assert_eq(pdf, gdf)
+
+
+def random_bool_column(size):
+    sz = bitmask_allocation_size_bytes(size)
+    data = np.random.randint(0, 255, dtype="u1", size=sz)
+    arr = np.random.randint(low=0, high=2, size=size).astype(np.bool)
+    return cudf.Series.from_masked_array(arr, data.view("i1"))
+
+
+def test_orc_bit_offsets():
+    np.random.seed(0)
+
+    gdf = cudf.DataFrame({"col_bool": random_bool_column(600000)})
```

Comment:
I am not so sure whether this will create the data-set that would always trigger failure.

Reply:
It should always create the same output, given np.random.seed(0). But the input could definitely be clearer. I'll change the way the mask is generated :+1:

Reply:
Ahh, I missed that.